x-pack/plugins/ingest_manager/server/services/epm/registry/requests.test.ts

```diff
@@ -53,13 +53,7 @@ describe('setupIngestManager', () => {
             throw new FetchError('message 3', 'system', { code: 'ESOMETHING' });
           })
           // this one succeeds
-          .mockImplementationOnce(() => Promise.resolve(new Response(successValue)))
-          .mockImplementationOnce(() => {
```

Comment:
Leaving these extra calls was messing up counts in later tests. Confirmed real behavior was expected. Assume issue is in clearing mocks, but decided to just drop these.